code/_onclick/hud/new_player.dm

```diff
@@ -348,14 +369,136 @@
 		set_button_status(FALSE)
 		return
 
-/atom/movable/screen/lobby/button/poll/update_overlays()
+/atom/movable/screen/lobby/button/bottom/poll/update_overlays()
 	. = ..()
 	if(new_poll)
 		. += mutable_appearance('icons/hud/lobby/poll_overlay.dmi', "new_poll")
 
-/atom/movable/screen/lobby/button/poll/Click(location, control, params)
+/atom/movable/screen/lobby/button/bottom/poll/Click(location, control, params)
 	. = ..()
 	if(!.)
 		return
 	var/mob/dead/new_player/new_player = hud.mymob
 	new_player.handle_player_polling()
+
+#define SHUTTER_MOVEMENT_DURATION 0.4 SECONDS
+#define SHUTTER_WAIT_DURATION 0.2 SECONDS
+
+/atom/movable/screen/lobby/button/collapse
+	name = "Collapse Lobby Menu"
+	icon = 'icons/hud/lobby/collapse_expand.dmi'
+	icon_state = "collapse"
+	base_icon_state = "collapse"
+	layer = LOBBY_COLLAPSE_BUTTON_LAYER
+	screen_loc = "TOP:-82,CENTER:-54"
+	always_shown = TRUE
+	var/matrix/ce_button_matrix = matrix()
+
+/atom/movable/screen/lobby/button/collapse/Click(location, control, params)
+	. = ..()
+	if(!.)
+		return
+	var/mob/dead/new_player/new_player = hud.mymob
```

Comment:
```suggestion
	var/mob/dead/new_player/new_player = hud.mymob
	var/inventory_status = newplayer.hud_used.inventory_shown
```

we can then use `inventory_status` to save some time, code room, and also just look nicer since it's more explicit

Reply:
could also do this for `hud_used` since we use that throughout this proc from what it looks like

Reply:
yeah uh i just remembered hud elements get a reference to their owner hud so we can just use that
i did make use of `inventory_status` cuz it looks nicer anyway
https://github.com/tgstation/tgstation/pull/76443/commits/0fb8e4ac86e78604a77158dd1b4b2aaf099c0b82